app/routes.js

```diff
@@ -1,9 +1,9 @@
-// const controller = require('./controllers/controller');
+const controller = require('./controllers/weets');
 const { healthCheck } = require('./controllers/healthCheck');
 
 exports.init = app => {
   app.get('/health', healthCheck);
-  // app.get('/endpoint/get/path', [], controller.methodGET);
+  app.get('/weet', controller.generateWeet);
```

Comment:
Same comment here, this will be used later, but is not needed for now.

In other hand, remember rest resources should generally be in plural, so `/weets`